test/e2e/lua/dynamic_configuration.go

```diff
@@ -39,6 +39,8 @@ import (
 var _ = framework.IngressNginxDescribe("Dynamic Configuration", func() {
 	f := framework.NewDefaultFramework("dynamic-configuration")
 
+	var defaultNginxConfigMapData map[string]string = nil
```

Comment:
where is this being used?

Reply:
Nice catch, issa relic of a previous iteration. Will nip